src/writer.h

```diff
@@ -51,6 +56,16 @@ class FileWriter : public Writer {
     virtual void write(const char* data, size_t len);
 };
 
+class LogWriter : public FileWriter {
+    LogLevel _logLevel;
+  public:
+    LogWriter(LogLevel logLevel = LOG_INFO) : FileWriter(Log::fd(), true) {
```

Comment:
This creates an implicit coupling with `Log::_file`, which lives its own life.
Currently, it's not a problem, since `Log` is always closed after the output file, but this dependency is fragile.
Alternatively, `LogWriter` could call `Log::writeRaw` or something.